btcjson/btcwalletextcmds.go

```diff
@@ -93,6 +93,84 @@ func NewRenameAccountCmd(oldAccount, newAccount string) *RenameAccountCmd {
 	}
 }
 
+// PsbtInput represents an input to include in the PSBT created by the
+// WalletCreateFundedPsbtCmd command.
+type PsbtInput struct {
+	Txid     string `json:"txid"`
+	Vout     int64  `json:"vout"`
+	Sequence int64  `json:"sequence"`
+}
+
+// PsbtOutput represents an output to include in the PSBT created by the
+// WalletCreateFundedPsbtCmd command.
+type PsbtOutput map[string]float64
+
+// NewPsbtOutput returns a new instance of a PSBT output to use with the
+// WalletCreateFundedPsbtCmd command.
+func NewPsbtOutput(address string, amount float64) PsbtOutput {
```

Comment:
The [documentation](https://bitcoincore.org/en/doc/0.20.0/rpc/wallet/walletcreatefundedpsbt/) in Bitcoin Core talks about a `data` field in psbt outputs. Looks like you forgot to add it.

```
       {                              (json object)
         "data": "hex",               (string, required) A key-value pair. The key must be "data", the value is hex-encoded data
       },
```

Reply:
Should I alias `PsbtOutput ` to an untyped `map[string]string` to support both cases? What's your advice on modelling heterogeneous lists like this?

Reply:
I aliased `PbstOutput` to `map[string]interface{}` which makes sense for both encoding and decoding both types of outputs.

Reply:
No strong opinion on this - `interface{}` is fine. There are other approaches, but that would be an overkill here. To me, it seems that the type is not concrete enough to deserve customized black magic.

@jakesyl Would you like to give your inputs here?

Reply:
missed this. I agree - `interface{}` should be fine here